src/beat_link_trigger/show_cues.clj

```diff
@@ -554,7 +554,7 @@
   Returns a tuple of the name by which it will be stored, and the
   content to be stored (or compared to see if it matches another cue)."
   [cue]
-   [(:comment cue) (dissoc cue :uuid :start :end :hue :linked :section)])
```

Comment:
As I study this more as part of implementing some new cue library features, I remember why I thought this was dangerous: it breaks assumptions about what is important when comparing cues. Since there are many other issues with this PR which have never been fixed, I am simply going to close it and try to implement the spirit of it correctly as part of my current enhancements.